src/main/java/org/zalando/nakadi/domain/SubscriptionListWrapper.java

```diff
@@ -0,0 +1,19 @@
+package org.zalando.nakadi.domain;
+
+import javax.annotation.concurrent.Immutable;
+import java.util.Collections;
+import java.util.List;
+
+@Immutable
+public class SubscriptionListWrapper {
```

Comment:
I would not like to touch pagination in this PR. 
I think pagination is a separate point that we should implement as a separate feature.